### PR TITLE
Move the player further into the room on traversal

### PR DESCRIPTION
**Pull request by 102113528**
This PR fixes the player being immediately teleported back into the previous room.

To test, walk through a bunch of doors and ensure the teleport back behaviour doesn't occur.

### CODE REVIEW

**Comment by 102113528** [comment]
Upon further investigation, this issue is related to door animations as discussed here: https://trello.com/c/EHOAWdDE/168-merlin-is-immediately-colliding-with-the-south-facing-door-when-going-through-the-north-door

Please don't merge this in until the door animation changes have been merged in!

**Review by BasilChristofis — COMMENTED** [review]
The new doors are in. I'll send this through. This looks like a simple change.

**Review by BasilChristofis — APPROVED** [review]
The new doors are in. I'll send this through. This looks like a simple change. Had wrong option selected on Review just now, It'll go through now.

**Pull request merged**
